app/models/card.rb

```diff
@@ -43,22 +43,23 @@ def set_review_interval_wrong_answers
     increment!(:incorrect_answers)
 
     if incorrect_answers == 3
-      update_attributes(review_date: Time.now + 12.hours,
+      update_attributes(review_date: Time.zone.now + 12.hours,
                         correct_answers: 0, incorrect_answers: 0)
     end
   end
 
-  def number_of_typos(answer)
-    DamerauLevenshtein.distance(original_text, answer, 0)
-  end
-
-
   def self.create_card_in_deck(user, params)
     deck = find_or_create_deck(params[:deck_name], user)
 
     deck.cards.build(params)
   end
 
+  def self.notify_user_for_review_pending_cards
+    Card.where('review_date <= ?', Time.zone.now).each do |card|
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Use `find_each` instead of `each`.